xAPI.md

```diff
@@ -2789,8 +2789,15 @@ See [Appendix G: Cross Domain Request Example](#AppendixG) for an example.
 
 ## 2.0 Data Storage and Retrieval
 
+This section contains implementation details and requirements surrounding the specifics of data storage within an 
+LRS and retrieval from that LRS.  As mentioned in the previous section, this communication is done through HTTP 
```

Comment:
Not sure about ```data storage within an LRS``` as this could be misread as how the data is arranged in the database. I'll try and think of some alternate wording; I understand what you mean and agree with the meaning.